portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,37 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.Random;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
-@WebServlet("/hello")
+import scorex.util.ArrayList;
+
+/** Handles requests sent to the /song URL. Try running a server and navigating to /hello! */
+@WebServlet("/songs")
 public class HelloWorldServlet extends HttpServlet {
 
+  private ArrayList<String> songs = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hi there!");
+    songs.add("\"Romantic Lover\" - Eyedress");
+    songs.add("\"88\" - Inner Wave");
+    songs.add("\"Trying Your Luck\" - The Strokes");
+
+    // had the servlet pick random song then send but tutorial specifies otherwise
```

Comment:
Nit: Please remove these comments if you're not planning on using them in your code. If you're planning on making edits to this in the future, consider adding a TODO(...) instead!